src/drivers/vga/text_mode.rs

```diff
@@ -6,32 +6,14 @@ use volatile::Volatile;
 use x86_64::instructions::port::{PortRead, PortWrite};
 
 use crate::drivers::serial;
+use crate::drivers::vga::ransid::RansidState;
 
 const TERMINAL_BUFFER: usize = 0xB8000;
 const WIDTH: usize = 80;
 const HEIGHT: usize = 25;
 
-#[repr(u8)]
-pub enum Color {
```

Comment:
You should move this over to `ransid.rs` even though it's unused for now, we can clean up some magic numbers there by using this enum